frontend/device/android/device.lua

```diff
@@ -107,6 +107,18 @@ local Device = Generic:new{
     should_restrict_JIT = true,
 }
 
+local BooxNovaPro = Device:new{
+    model = "Onyx_Boox_NovaPro",
+    hasNaturalLight = yes,
+    hasLightLevelFallback = no,
+    frontlight_settings = {
+        frontlight_white = "/sys/class/backlight/white",
+        frontlight_red = "/sys/class/backlight/warm",
```

Comment:
use `frontlight_mixer =  "/sys/class/backlight/warm"` instead

Reply:
On mixer devices does writing to the `/sys/class/backlight/x/color` change the backlight colour or change the brightness of a separate set of LEDs?

The two brightness files on the Nova Pro control independent sets of LEDs.

Reply:
We don't do that but probably the system does something like that (ie: adjust brightness intensity based on warmth level). See https://github.com/koreader/koreader/issues/4291#issuecomment-439700537.

Reply:
I feel like I cannot help because I don't know nothing about your device. Do you found some issues with your current implementation?